tools/who_what_benchmark/requirements.txt

```diff
@@ -1,9 +1,11 @@
+--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
 transformers>=4.35.2
 sentence-transformers>=2.2.2
-openvino>=2024.3.0
+openvino
+openvino-tokenizers
+openvino-genai
 openvino-telemetry
 optimum-intel>=1.19.0
```

Comment:
I see that CI uses latest optimum-intel version https://github.com/openvinotoolkit/openvino.genai/blob/e6a435fdf4b4c14d0c664aaf7fe1d763b341faff/.github/workflows/llm_bench-python.yml#L105. Should we reflect it in requirements.txt ?

Reply:
This will break the example https://github.com/openvinotoolkit/nncf/tree/develop/examples/llm_compression/openvino/tiny_llama_find_hyperparams in NNCF.